common/src/main/java/com/microsoft/identity/common/internal/request/AdalBrokerRequestAdapter.java

```diff
@@ -121,8 +125,18 @@ public BrokerAcquireTokenOperationParameters brokerInteractiveParametersFromActi
                 AuthenticationConstants.Broker.ACCOUNT_CLIENTID_KEY)
         );
 
+        // V1 Broker would compute the redirect_uri for the calling package, rather than
+        // 'trust' the provided value -- this had the unfortunate consequence of allowing
+        // callers to pass non-URL-encoded signature hashes into the library despite the documentation
+        // prescribing otherwise. The ADAL.NET implementation unfortunately RELIES on this behavior,
+        // forcing customers to use non-encoded values in order to pass validation check inside of
+        // ADAL.NET. In order to not regress this experience, the redirect URI must now be computed
+        // meaning that the ACCOUNT_REDIRECT parameter is basically ignored.
```

Comment:
might be worth considering getting rid of ACCOUNT_REDIRECT entirely (from MSAL/Broker/Common)

Reply:
Thanks @rpdome -- I think this is the right call. I'll exclude those changes from this PR for now and will include them in a follow up